### DISCUSSION TITLE
Persist sidebarFullyCollapsibleOnDesktop between requests

### DISCUSSION THREAD

**Discussion by cirrion-software** [filamentphp/filament#11810] [Help]
### Package

Panel builder

### Package Version

v3.2.47

### How can we help you?

Is there a way to persist the collapsed sidebar state between requests. I am using **sidebarFullyCollapsibleOnDesktop** but if you collapse the sidebar it is expanded on the next request. Would be nice to stay collapsed until manually expanded.

**Comment by micraux** [comment]
You could store the collapsed sidebar state in session
